joblib/_parallel_backends.py

```diff
@@ -40,9 +40,12 @@ def __init__(self, nesting_level=None, inner_max_num_threads=None):
 
     MAX_NUM_THREADS_VARS = [
         'OMP_NUM_THREADS', 'OPENBLAS_NUM_THREADS', 'MKL_NUM_THREADS',
-        'BLIS_NUM_THREADS', 'VECLIB_MAXIMUM_THREADS', 'NUMEXPR_NUM_THREADS'
+        'BLIS_NUM_THREADS', 'VECLIB_MAXIMUM_THREADS', 'TBB_NUM_THREADS',
```

Comment:
Is `TBB_NUM_THREADS` kind of dummy variable? Because there is no such control variable in TBB itself. @ogrisel

Reply:
Oops indeed, this is a left over I wanted to remove. thanks for the catch.